apps/graphql/src/apps/booking/Booking.js

```diff
@@ -83,3 +96,22 @@ export type Booking = {|
   +type: 'BookingReturn' | 'BookingMulticity' | 'BookingOneWay',
   ...TypeSpecificData,
 |};
+
+export type LanguageInput = {|
+  language: string,
```

Comment:
Read only with `+`?

Reply:
Ooops yep >__<! done, fixed the other ones as well (`HoldBags`, etc..)